### DISCUSSION TITLE
Nightly build no longer nightly?

### DISCUSSION THREAD

**Discussion by subnut** [neovim/neovim#13582] [Q&A] [answered]
The last nightly build was released **4** days ago. What's going on?

**Comment by ccchapman** [comment, marked as answer]
Builds were ported to GitHub Actions recently. The [most recent nightly builds have failed](https://github.com/neovim/neovim/actions?query=workflow%3ARelease). The [issue for this migration](https://github.com/neovim/neovim/issues/13312) is still open and ongoing, so I suspect things should be resolved once the migration is complete.
